cmd/kpod/main.go

```diff
@@ -7,14 +7,17 @@ import (
 	"github.com/urfave/cli"
 )
 
+const VERSION string = "0.0.1"
```

Comment:
We need to eventually merge this with cri-o version, they should be the same, but this is fine for now.
Constants should be Capatalized but not Upper Case I believe?